Translation corrections (#251)
07th-mod/higurashi-console-arcs: Update/yoig_tips_010.txt

//r本当は、彼女だってそろそろ、気づいているはずだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "本当は、彼女だってそろそろ、気づいているはずだ。",

> 			NULL, "The truth is, even she should be realizing it sooner or later.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The truth is, even she should realize it sooner or later.", Line_Normal);
```

//r彼女を少しでも楽にしたいと発起して、歯を食いしばってがんばり続けて、ようやくのことで『これ』を手に入れたってのに、…俺は、彼女に見せることができなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "彼女を少しでも楽にしたいと発起して、歯を食いしばってがんばり続けて、ようやくのことで『これ』を手に入れたってのに、…俺は、彼女に見せることができなかった。",

> 			NULL, "Even though I'd resolved myself to make things easier for her and finally obtained <i>this</i> after shedding blood and tears of effort... I couldn't show it to her.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Even though I'd resolved myself to make things easier for her and finally obtained <i>this</i> after shedding blood and tears... I couldn't show it to her.", Line_Normal);
```

//rさっきからアルコールを取り続けても、…これを見せることで彼女から下される審判を聞くことの恐怖に耐えられなくて、俺はどうしても、覚悟を決めることができなかった…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "さっきからアルコールを取り続けても、…これを見せることで彼女から下される審判を聞くことの恐怖に耐えられなくて、俺はどうしても、覚悟を決めることができなかった…。",

> 			NULL, "Even if I keep drinking like I have been these days... I still can't bring myself to gather the courage to show her... terrified by the prospect of losing her love...", Line_Normal);

Reviewer: ```suggestion
			NULL, "Even if I keep drinking like I have been these days... I still can't bring myself to gather the courage to show her... I'm terrified by the prospect of losing her love...", Line_Normal);
```
